lib/main_micro_alp.pm

```diff
@@ -0,0 +1,280 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: module loader for MicroOS, SLE Micro, Leap Micro and ALP.
+# Maintainer: qa-c@suse.de
+
+package main_micro_alp;
+use strict;
+use warnings;
+use base 'Exporter';
+use Exporter;
+use main_common;
+use main_containers qw(load_container_tests is_container_test);
+use testapi qw(check_var get_required_var get_var set_var);
+use version_utils qw(is_microos is_sle_micro is_leap_micro is_alp is_staging is_released is_transactional is_rt);
+use utils;
+use Utils::Architectures;
+use Utils::Backends;
+
+our @EXPORT = qw(
+  load_tests
+);
```

Comment:
Since you're doing static calls, you can even remove this, [along with all the exporter stuff](https://gist.github.com/foursixnine/5f001c5f924e077a713383cfdd9dd0ba) and it will still work (don't believe me 100% though)

Reply:
it works, thanks!